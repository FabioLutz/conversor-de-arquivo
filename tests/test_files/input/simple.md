# Divisores

---
***
___

# Títulos

# Título (H1)
## Subtítulo (H2)
### Subtítulo menor (H3)
#### Subtítulo menor ainda (H4)
##### Subtítulo muito menor ainda (H5)
###### Subtítulo o menor (H6)

# Formatações

**Texto em negrito**

_Texto em itálico_

***Texto em negrito e itálico***

# Listas

* Lista com asterisco
+ Lista com sinal de adição
- Lista com hífen

## Listas não ordenadas

- Lista não ordenada
    - Subitem
        - Sub-subitem
    - Outro subitem

## Listas ordenadas

1. Lista ordenada
    1. Subitem
        1. Sub-subitem
    2. Subitem 2
    7. Número 7

# Citação

> Citação
>
>> Essa é uma citação
>
> Fim da citação

# Links

<https://pt.wikipedia.org/wiki/Markdown>

<user@example.com>

[Link](https://pt.wikipedia.org/wiki/Markdown "Wikipédia Markdown")

[Link por referência][1]

[1]: <https://pt.wikipedia.org/wiki/Markdown> "Wikipédia Markdown"

# Código

`print("Olá, mundo!")`
