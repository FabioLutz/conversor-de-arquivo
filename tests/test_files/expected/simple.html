<h1>Divisores</h1>
<hr />
<hr />
<hr />
<h1>Títulos</h1>
<h1>Título (H1)</h1>
<h2>Subtítulo (H2)</h2>
<h3>Subtítulo menor (H3)</h3>
<h4>Subtítulo menor ainda (H4)</h4>
<h5>Subtítulo muito menor ainda (H5)</h5>
<h6>Subtítulo o menor (H6)</h6>
<h1>Formatações</h1>
<p><strong>Texto em negrito</strong></p>
<p><em>Texto em itálico</em></p>
<p><strong><em>Texto em negrito e itálico</em></strong></p>
<h1>Listas</h1>
<ul>
<li>Lista com asterisco</li>
<li>Lista com sinal de adição</li>
<li>Lista com hífen</li>
</ul>
<h2>Listas não ordenadas</h2>
<ul>
<li>Lista não ordenada<ul>
<li>Subitem<ul>
<li>Sub-subitem</li>
</ul>
</li>
<li>Outro subitem</li>
</ul>
</li>
</ul>
<h2>Listas ordenadas</h2>
<ol>
<li>Lista ordenada<ol>
<li>Subitem<ol>
<li>Sub-subitem</li>
</ol>
</li>
<li>Subitem 2</li>
<li>Número 7</li>
</ol>
</li>
</ol>
<h1>Citação</h1>
<blockquote>
<p>Citação</p>
<blockquote>
<p>Essa é uma citação</p>
</blockquote>
<p>Fim da citação</p>
</blockquote>
<h1>Links</h1>
<p><a href="https://pt.wikipedia.org/wiki/Markdown">https://pt.wikipedia.org/wiki/Markdown</a></p>
<p><a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#117;&#115;&#101;&#114;&#64;&#101;&#120;&#97;&#109;&#112;&#108;&#101;&#46;&#99;&#111;&#109;">&#117;&#115;&#101;&#114;&#64;&#101;&#120;&#97;&#109;&#112;&#108;&#101;&#46;&#99;&#111;&#109;</a></p>
<p><a href="https://pt.wikipedia.org/wiki/Markdown" title="Wikipédia Markdown">Link</a></p>
<p><a href="https://pt.wikipedia.org/wiki/Markdown" title="Wikipédia Markdown">Link por referência</a></p>
<h1>Código</h1>
<p><code>print("Olá, mundo!")</code></p>